packages/manager/apps/dedicated/client/app/billing/confirmTerminate/legacy/termination-legacy.controller.js

```diff
@@ -0,0 +1,99 @@
+import get from 'lodash/get';
+
+export default class TerminateServiceCtrl {
+  /* @ngInject */
```

Comment:
Didn't update the code (as its legacy) only changed this part to export and inject dependencies